.github/workflows/blade-chromatic.yml

```diff
@@ -18,7 +18,11 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 18.12.1
+          # https://stackoverflow.com/a/73824182
+          registry-url: "https://npm.pkg.github.com"
```

Comment:
what does this solve?

Reply:
This was related to i18nify package's default registry being npm.pkg.github in our lockfile, without this + the NODE_AUTH_TOKEN the installation fails on CI